proto/viam/app/mltraining/v1/ml_training.proto

```diff
@@ -56,6 +56,7 @@ message TrainingJobMetadata {
   google.protobuf.Timestamp created_on = 3;
   google.protobuf.Timestamp last_modified = 4;
   string synced_model_id = 5;
+  string user_email = 6;
```

Comment:
Is this all you need for checking the user state? Is there a reason to choose this over a user ID?

Reply:
I chose email because `web.UserInfo` contains an [email field](https://github.com/viamrobotics/goutils/blob/01921ff36688a2243476ce57f806bec484cfac7f/web/user.go#L17), and this is what's used to represent a user when constructing the `[auth.State](https://github.com/viamrobotics/app/blob/main/auth/state.go#L16)`.